src/index.js

```diff
@@ -195,17 +198,15 @@ class CreditCardInput extends Component<Props, State> {
   };
 
   handleCardExpiryChange = (e: SyntheticInputEvent<*>) => {
-    const cardExpiry = e.target.value;
-    const cardExpiryLength = cardExpiry.split(' / ').join('').length;
+    const cardExpiry = e.target.value.split(' / ').join('/');
     payment.formatCardExpiry(document.getElementById('card-expiry'));
 
     this.setFieldValid();
-    if (cardExpiryLength >= 4) {
-      if (payment.fns.validateCardExpiry(cardExpiry)) {
-        this.cvcField.focus();
-      } else {
-        this.setFieldInvalid('Expiry date is invalid');
-      }
+    const expiryError = creditCardExpiryValidator.isExpiryInvalid(cardExpiry);
```

Comment:
Can we perhaps keep the `cardExpiryLength` assertion? Otherwise, an error will always display as you're typing in an expiry.

Reply:
Wow, that was a good catch! Indeed, looks better now (, even UX-wise).